src/API/Notes.php

```diff
@@ -77,6 +77,32 @@ public function register_routes() {
 				'schema' => array( $this, 'get_public_item_schema' ),
 			)
 		);
+
+		register_rest_route(
```

Comment:
I didn't check the full PR to see if you had done this already, but we need to have unit test coverage on these new endpoints. At a minimum it is nice to have a test to assert that users without proper permissions can not use these new endpoints, and ones that verify the deletes happen as expected.

Reply:
I added some unit tests in the [PR 4320 (Inbox notification: add event recording)](https://github.com/woocommerce/woocommerce-admin/pull/4320).
They cover all the processes but I missed the permission check you mentioned.
This was added in the commit [f4b62e2](https://github.com/woocommerce/woocommerce-admin/pull/4320/commits/f4b62e26d9fd84e3ccfe6eb056a543da0c3c1ff3)
Thank you Timmy!